react/features/base/react/components/web/InlineDialogFailure.js

```diff
@@ -0,0 +1,72 @@
+import AKButton from '@atlaskit/button';
+import React, { Component } from 'react';
+
+import { translate } from '../../../i18n';
+
+declare var interfaceConfig: Object;
+
+/**
+ * Inline dialog the represents a failure and allows a retry.
+ */
+class InlineDialogFailure extends Component {
```

Comment:
Hm, in my head it was the content of the inline dialog that represents failure. That's what we render, so I don't want to name it XXXDialog, because it's not the dialog itself..I have other doubts about this one, because I wonder if it's generic enough and it can be put in something else than InlineDialog as well...